app/views/volunteers/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= simple_form_for volunteer, url: '/#apply', method: :post do |f| %>
+  <%= f.input :country, collection: countries, prompt: "Choose your country", label: false %>
+
+  <%= f.input :state, label: false, collection: [], input_html: {hidden: true} %>
+
+  <%= f.input :city, label: false, collection: [], input_html: {hidden: true} %>
+
+  <%= f.input :name, label: false, required: true, input_html: {oninvalid: "this.setCustomValidity('What is your name?')", oninput: "setCustomValidity('')"} %>
+
+  <%= f.input :email, label: false, input_html: { autocomplete: 'email' }, input_html: {minlength: 7, oninvalid: "this.setCustomValidity('What is your email address?')", oninput: "setCustomValidity('')"} %>
+
+  <%= f.input :phone, label: false, input_html: { autocomplete: 'tel' }, input_html: {minlength: 7, oninvalid: "setCustomValidity('What is your phone number?')", oninput: "setCustomValidity('')"} %>
+
+  <%= f.input :group, collection: volunteer_support_units, prompt: "Choose what can you help with", label: false %>
+
+  <%= f.input :about, label: false, as: :text, input_html: {minlength: 20, oninvalid: "setCustomValidity('How would you like to help?')", oninput: "setCustomValidity('')"} %>
+
+  <%= f.input :immunity, as: :radio_buttons, :collection => [ ['Yes', true], ['No', false] ], :checked => ['No', false] %>

```

Comment:
@hantuzun I've been wondering: how does immunity work for COVID-19? Is anyone immune? Does that just apply to people that have already had it?

Reply:
My suggestion is to put a "?" icon there, and link it to a help box with text on how you determine that, noting (for now) that it requires and answer.
I'd do the same with the "Immunity required" boxes, link 'em to help text (which still needs to be written

Reply:
@kopchickm Immunity is only through recovery. 

@Donnadieu I agree. We should have a FAQ page entry on that.